src/main/kotlin/com/bridgecrew/utils/Constants.kt

```diff
@@ -46,3 +46,10 @@ val SUPPRESSION_BUTTON_ALLOWED_FILE_TYPES: Set<FileType> = setOf(
         FileType.YAML,
         FileType.TERRAFORM
 )
+
+// license
+const val VIOLATED_LICENSES_TITLE = "Non-Complaint license"
+const val UNKNOWN_LICENSES_TITLE = "Unknown license"
+
+const val VIOLATED_LICENSES_DESCRIPTION = "At least one of the license(s) used by this package and/or its dependencies are not OSI approved."
```

Comment:
worth using these constants also in error bubble (I also added them there under ErrorBubbleInnerPanel)

Reply:
fixed :)